scripts/ci/docker-compose/backend-mssql.yml

```diff
@@ -20,12 +20,12 @@ services:
   airflow:
     environment:
       - BACKEND=mssql
-      - AIRFLOW__CORE__SQL_ALCHEMY_CONN=mssql+pyodbc://sa:Airflow123@mssql:1433/master?driver=ODBC+Driver+17+for+SQL+Server
-      - AIRFLOW__CELERY__RESULT_BACKEND=db+mssql+pyodbc://sa:Airflow123@mssql:1433/master?driver=ODBC+Driver+17+for+SQL+Server
+      - AIRFLOW__CORE__SQL_ALCHEMY_CONN=mssql+pyodbc://sa:Airflow123@mssql:1433/airflow?driver=ODBC+Driver+17+for+SQL+Server
+      - AIRFLOW__CELERY__RESULT_BACKEND=db+mssql+pyodbc://sa:Airflow123@mssql:1433/airflow?driver=ODBC+Driver+17+for+SQL+Server
       - AIRFLOW__CORE__EXECUTOR=LocalExecutor
     depends_on:
-      mssql:
-        condition: service_healthy
+      - mssqlsetup
```

Comment:
What do you think about add condition - `service_completed_successfully`?
https://github.com/apache/airflow/blob/14b3e66ad5e2325eb2ee7b9c6661ff7b636d017d/docs/apache-airflow/start/docker-compose.yaml#L114-L115

Reply:
What do you think about it? 

when I try to use it in IntelliJ it tells me it's not expected here. Can you tell me whether it is a good idea to use it? Is it supported / usable in all compose versions?

Reply:
`service_healty` works - but when I tried to replace `service_healthy` with `service completed successfully` - it hangs indefinitely, but maybe this is something I am doing wrong - if you could check how to make use of it, it would be great

Reply:
By using this condition, the container will not be started until another container is stopped, so you can do additional initiations steps before running the main container, e.g. initialize the database, get dependencies and then start the main container.
Here are docs about it:
https://github.com/compose-spec/compose-spec/blob/master/spec.md#long-syntax-1
This is supported starting with the 1.29.0 (2021-04-06) version. https://docs.docker.com/compose/release-notes/#1290
Here is another real-life example with node app:
https://github.com/KlubJagiellonski/pola-web/blob/8c93e0b79231c2b92a984099df3209e287ac9458/docker-compose.yaml#L17-L18

Reply:
I do not know basically how to make dependency on two services - one with a condition and one without  - it does not seem to be supported to depend on one service that has no condition and another service that depends on "just service" - the first one requires dict, the second list, but I could not find a way how to make both work.

that 's why I made `airflow` depends on both mssql and mssqlsetup (unconditionally) but also make mssqlsetup depends conditionally on healthy status of mssql 

I think it does the job nicely - but if you can find a better solution - I am all ears.

Reply:
Ah - I see, I think I figured it out :)